File: android/app/src/main/java/bjc/shoperp/domain/Shop.java
package bjc.shoperp.domain;

import java.util.Date;

/**
 * Created by hcq on 2018/2/20.
 */

public class Shop {
    public long Id ;

    public int PopType ;

    public String PopSellerId ;

    public String PopSellerNumberId ;

    public String AppKey ;

    public String AppSecret ;

    public String AppAccessToken ;

    public String AppRefreshToken;

    public String AppCallbackUrl ;

    public String Mark;

    public float CommissionPer ;

    public Date CreateTime ;

    public Date UpdateTime ;

    public String LastUpdateOperator ;

    public boolean Enabled ;

    public int ShippingHours ;

    public int FirstDeliveryHours ;

    public int SecondDeliveryHours ;

    public boolean AppEnabled ;
}

File: net/ShopErp.Domain/Shop.cs
using System;

namespace ShopErp.Domain
{
    public class Shop
    {
        public long Id { get; set; }

        public PopType PopType { get; set; }

        public string PopSellerId { get; set; }

        public string PopSellerNumberId { get; set; }

        public string PopTalkId { get; set; }

        public string AppKey { get; set; }

        public string AppSecret { get; set; }

        public string AppAccessToken { get; set; }

        public string AppRefreshToken { get; set; }

        public string AppCallbackUrl { get; set; }

        public string Mark { get; set; }

        public float CommissionPer { get; set; }

        public DateTime CreateTime { get; set; }

        public DateTime UpdateTime { get; set; }

        public string LastUpdateOperator { get; set; }

        public bool Enabled { get; set; }

        public int ShippingHours { get; set; }

        public int FirstDeliveryHours { get; set; }

        public int SecondDeliveryHours { get; set; }

        public bool AppEnabled { get; set; }

        public bool WuliuEnabled { get; set; }
    }
}
